### DISCUSSION TITLE
FlexBox and minPxPerSec Renderering Issue

### DISCUSSION THREAD

**Discussion by bandell07** [katspaugh/wavesurfer.js#3734] [Q&A]
I am currently trying to upgrade from v6 to v7. I currently have the wavesurfer in a flex box with flex-grow:1 and am using the scrollParent property. I see that was replaced by using the minPxPerSec property, once I did that the end of the waveform started jumping constantly. Here is an example of what I'm experiencing, https://jsfiddle.net/velvetthunder07/wczq9p04/19/. I saw this issue, https://github.com/katspaugh/wavesurfer.js/issues/2945, which sounds similar to what I'm experiencing however the fix doesn't appear to be working. 

Is there some other property or something that I need to set to get this to work properly?

Thanks!

**Comment by awallish** [comment]
I am experience the same thing. Did you figure out the solution?

**Reply by bandell07** [reply]
Unfortunately I did not. I meant to come back and check out the source to try and figure it out but life. If I find a solution I'll post it here.
